server/serverAPI/controller/BaseController.ts

```diff
@@ -16,21 +17,15 @@ export default class BaseController {
         let logs = await object.validate()
 
         if (logs.length > 0) {
-            return Promise.reject(this.sendError(400, res, logs));
+            return Promise.reject(this.send(400, res, logs));
         }
 
         return object;
     }
 
-    protected sendError(statusCode: number, res: Response, message?: any): Response<any, Record<string, any>> {
+    protected send(statusCode: ResponseCodes, res: Response, message?: any): Response<any, Record<string, any>> {
         return res.status(statusCode)
-            .json(ResponseFormatter.formatAsJSON(ResponseTypes.ERROR, message));
-    }
-
-    protected sendSuccess(statusCode: number, res: Response, message?: any)
-        : Response<any, Record<string, any>> {
-        return res.status(statusCode)
-            .json(ResponseFormatter.formatAsJSON(ResponseTypes.SUCCESS, message));
+            .json(ResponseFormatter.formatAsJSON(message));
```

Comment:
Since it looks like `.formatAsJSON` is now a no-op, can you just do `.json(message)`?